www/include/configuration/configObject/contact/formContact.php

```diff
@@ -784,6 +784,11 @@ function myReplace()
 
 if ($form->validate() && $from_list_menu == false) {
     $cctObj = $form->getElement('contact_id');
+    if (!$centreon->user->admin && $contactId) {
```

Comment:
$contactId is never set... please can you rename all $centreon_id with $centreonId

Reply:
contactId is set in contact.php, otherwise whole edit functionality would have been broken.